curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-cash-register-project/cash-register.md

```diff
@@ -1,252 +1,559 @@
 ---
 id: aa2e6f85cab2ab736c9a9b24
 title: Build A Cash Register
-challengeType: 5
+challengeType: 14
 forumTopicId: 16012
 dashedName: build-a-cash-register
 ---
 
 # --description--
 
-Design a cash register drawer function `checkCashRegister()` that accepts purchase price as the first argument (`price`), payment as the second argument (`cash`), and cash-in-drawer (`cid`) as the third argument.
+Here you'll build a cash register app that will return change to the customer based on the price of the item, the amount of cash provided by the customer, and the amount of cash in the cash drawer. You'll also need to show different messages to the user in different scenarios, such as when the customer provides too little cash or when the cash drawer doesn't have enough to issue the correct change.
 
-`cid` is a 2D array listing available currency.
+There are a few variables you'll need to use in your code:
 
-The `checkCashRegister()` function should always return an object with a `status` key and a `change` key.
+- `price`: the price of the item as a floating point number
+- `cash`: is amount of cash provided by the customer for the item, which is provided via an `input` element on the page
+- `cid`: cash-in-drawer, a 2D array listing available currency in the cash drawer
```

Comment:
```suggestion
- `price`: the price of the item as a floating-point number.
- `cash`: the amount of cash provided by the customer for the item, which is provided via an `input` element on the page.
- `cid`: cash-in-drawer, a 2D array listing available currency in the cash drawer.
```

Reply:
Thanks for catching these issues, too, @Sembauke.

The one thing I may go back in change is "floating-point number". It seems like we use the unhyphenated version, "floating point number" in other parts of the curriculum, but I can't find the hyphenated version anywhere.

I'll do more digging and possibly change it to "floating point number" for consistency.